dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/LaunchArguments.java

```diff
@@ -322,6 +322,8 @@ private static String commandLineToInternalPhaseName(String phaseName) {
                 return CheckpointPhase.AFTER_APP_START.toString();
             case "beforeappstart":
                 return CheckpointPhase.BEFORE_APP_START.toString();
+            case "apprequested":
```

Comment:
Copyright date looks wrong on this file too? I'm not going to check them all ;-)